crypto/ct/ct_log.c

```diff
@@ -7,6 +7,12 @@
  * https://www.openssl.org/source/license.html
  */
 
+/*
+ * SHA256 low level APIs are deprecated for public use, but still ok for
+ * internal use.
+ */
+#include "internal/deprecated.h"
```

Comment:
This file seems to be an example where we *should* convert to using EVP.

Reply:
Converted to EVP.